### ISSUE TITLE
Top to Bottom?

### ISSUE THREAD

**Issue by cmdrlanc73** [hexchat/hexchat#1743]
Is there a way to make the text lines appear top to bottom, rather than new line on the page of screen?

**Comment by TingPing** [comment]
Duplicate of #903

**Comment by cmdrlanc73** [comment]
This not a duplicate. The cited number has nothing to do with my question. I want to have the new text at the top. This cited #903 is about a bug where there is only one line at the top of the window.

**Comment by TingPing** [comment]
So you want it to 'loop'?

**Comment by cmdrlanc73** [comment]
No. I want each new line of text to appear at the top and push the line before it further down the page.
